src/components/gridView/gridView.api.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "GridView",
+  "category": "layoutsAndTemplates",
+  "description": "A auto-generated grid view that calculate item size according to given props",
```

Comment:
An

Reply:
Done. 
I also formatted the `hint.api.json` file